.github/scripts/execute_single_benchmark.sh

```diff
@@ -64,8 +64,8 @@ elif [ "$LOADER_NAME" == "colmap-loader" ]; then
     --max_resolution ${MAX_RESOLUTION} \
     ${SHARE_INTRINSICS_ARG}
 
-elif [ "$LOADER_NAME" == "astronet" ]; then
-  python gtsfm/runner/run_scene_optimizer_astronet.py \
+elif [ "$LOADER_NAME" == "astrovision" ]; then
```

Comment:
What's the difference between astronet and astrovision?

Reply:
They're the same I just changed the name in the final publication